bin/aursearch

```diff
@@ -164,15 +160,12 @@ if ((!$#)); then
     exit 1
 fi
 
-case $searchby in
-  "name-desc") urls=("$*") ;;&
-  "name"|"maintainer") urls=("$@") ;;&
-  "name"|"maintainer"|"name-desc")
-    { for u in "${urls[@]}"; do
-      echo "$aurweb/rpc/?v=5&type=search&by=$searchby&arg=${u/ /%20}"
-    done } | getjson | parse "$format"
-  ;;
-  *) match "$list" "$@" | split | getjson | parse "$format"
-esac
+if [ -n "$searchby" ]; then
+  { for q in "$@"; do
+    echo "$aurweb/rpc/?v=5&type=search&by=$searchby&arg=${q/ /%20}"
```

Comment:
Since this bypasses the split function, it doesn't URL encode `@` and `+`. A better solution may be to generalize `split` to take arguments, and add URL encoding for the space character. e.g.
```
    declare type=$1 by=$2
    declare rpc="$aurweb/rpc/?v=5&type=$type"

    if [[ -n $by ]] && [[ $by != none ]]; then
        rpc+="&by=$by"
    fi

(...)
        gsub(" ","%20")
```